manualtesting/manual_testing.go

```diff
@@ -149,21 +149,19 @@ func manualTest(c *web.Context, w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-func getChannelID(a *app.App, channelname string, teamid string, userid string) (id string, err bool) {
+func getChannelID(a *app.App, channelname string, teamid string, userid string) (string, bool) {
 	// Grab all the channels
-	result := <-a.Srv.Store.Channel().GetChannels(teamid, userid, false)
-	if result.Err != nil {
+	data, err := a.Srv.Store.Channel().GetChannels(teamid, userid, false)
```

Comment:
probably here we can use a better name for the variable, probably `channels`.

Reply:
Done